assets/js/src/newsletter_editor/blocks/woocommerceContent.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable func-names */
```

Comment:
WHY? I thought we want to remove those and not add more 😞

Reply:
I removed that comment and fixed the ESLint warnings.